styles/responsive_styles.h

```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
                              TR2>,
                     EFFECT_CLASH>;
 
-// ResponsiveBlastL<BLAST COLOR, TRANSITION1, TRANSITION2, TOP, BOTTOM, SIZE>
-// Implements Blast effect that will move based on angle of the blade instead of random location
-// TRANSITION1 & TRANSITION2 = transition Begin & End
+// ResponsiveBlastL<BLAST COLOR, FADEOUT_MS, WAVE_SIZE, WAVE_SPEED, TOP, BOTTOM, EFFECT>
+// Implements Blast effect that will move based on angle of the blade instead of random location Blast will impact and disperse along the blade from original position
+// FADE = fade time ms
+// WAVE_SIZE = size
+// WAVE MS = speed ms
 // TOP = uppermost Blast position limit, BOTTOM = lowermost Blast position limit, 32768 = tip, 0 = hilt
-// SIZE controls blast size 0 ~ 32768
+
 template<class COLOR,
-         class TR1 = TrInstant,
-         class TR2 = TrFade<200>,
+         int FADE = 400,
+         int SIZE = 100,
+         int SPEED = 400,
          int TOP = 28000,
          int BOTTOM = 8000,
-         int SIZE = 8000>
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
 using ResponsiveBlastL =
-  MultiTransitionEffectL<TrConcat<TR1,
-                                  AlphaL<COLOR, Bump<Scale<BladeAngle<>,Int<TOP>,Int<BOTTOM>>,Int<SIZE>>>,
-                                  TR2>,
-                         EFFECT_BLAST>;
+         AlphaL<MultiTransitionEffectL<
+                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, Int<TOP>, Int<BOTTOM>>>,EFFECT>, 
+                  Bump<Scale<BladeAngle<>,Int<TOP>,Int<BOTTOM>>,Int<24000>>>;
+
+// ResponsiveBlastWaveL<BLAST COLOR, FADEOUT_MS, WAVE_SIZE, WAVE_SPEED, TOP, BOTTOM>
+// Implements Blast effect that will move based on angle of the blade instead of random location Blast will impact and split up and down the length of the blade from original position
+// FADE = fade time ms
+// WAVE_SIZE = size
+// WAVE MS = speed ms
+// TOP = uppermost Blast position limit, BOTTOM = lowermost Blast position limit, 32768 = tip, 0 = hilt
+
+template<class COLOR,
+         int FADE = 400,
+         int SIZE = 100,
+         int SPEED = 400,
+         int TOP = 28000,
+         int BOTTOM = 8000,
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
+using ResponsiveBlastWaveL =
+         MultiTransitionEffectL<
+                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, 
+                           Int<TOP>, 
```

Comment:
The way this is indented, it looks like an argument to TrWaveX, but it's an argument to Scale<>, please fix.